### PR TITLE
fix(Creature): Remove bad SmartAI entries.

### PR DESCRIPTION
**Pull request by heyitsbench**
## Changes Proposed:

This PR proposes changes to:
-  [X] Database (SAI, creatures).

## Issues Addressed:

- Closes unreported issue.

## Tests Performed:

This PR:
- [X] Requires no further testing.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Nyeriah** [comment]
Some context?

**Comment by heyitsbench** [comment]
These creatures have smart scripts that only whisper a blank line to an invoker. That is the only thing their scripts do. They are not Blizzlike, so I propose they be removed.

File: data/sql/updates/pending_db_world/bad-ai.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+UPDATE `creature_template` SET `AIName` = '' WHERE `entry` IN (2307, 11940, 11941, 11943, 19064, 32743, 37072);
+
+DELETE FROM `smart_scripts` WHERE `entryorguid` IN (2307, 11940, 11941, 11943, 19064, 32743, 37072);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/bad-ai.sql:3]: You need to delete with SourceGroup

Comment (heyitsbench) [reply data/sql/updates/pending_db_world/bad-ai.sql]: Did you mean `source_type`? If so, done.

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/bad-ai.sql]: Sure, SourceGroup must be conditions. Cant remember everything while on the road x)

**Comment by sudlud** [comment]
just tracked this down, SAI for 37072 was added with #1570

https://github.com/azerothcore/azerothcore-wotlk/pull/1570/files#diff-45c8d042dba1864ec4c630065755e6a365964d113388ea0035c1fb71bc0f2517R851

not sure if this helps to understand what this was initially meant to be used for

**Comment by avarishd** [comment]
Wasn't this the thing that was causing crashes when it was being played?

**Comment by heyitsbench** [comment]
37072 is a creature in Orgrimmar, if that is somehow causing games to crash from an RP event in Nagrand, I'd be quite surprised.

**Review by Nyeriah — APPROVED** [review]

**Comment by sudlud** [comment]
Yeah, looks like this was added in the linked PR unintentionally.

**Pull request merged**